index.bs

```diff
@@ -715,6 +716,13 @@ A [=sensor=] has an associated <dfn>periodic reporting mode flag</dfn> which is
 
 A [=sensor=] has an associated <dfn>current sampling frequency</dfn> which is initially null.
```

Comment:
sampling frequency is a frequency at which UA obtains sensor readings from the underlying platform